src/resources/ha-style.ts

```diff
@@ -157,6 +157,8 @@ documentContainer.innerHTML = `<custom-style>
       --state-device_tracker-home-color: var(--green-color);
       --state-fan-active-color: var(--cyan-color);
       --state-humidifier-active-color: var(--blue-color);
+      --state-humidifier-humidify-color: var(--blue-color);
+      --state-humidifier-dry-color: var(--orange-color);
```

Comment:
State color doesn't apply to action, only state.
https://www.home-assistant.io/integrations/frontend/#state-color

We should rename `--state-humidifier-active-color` with `--state-humidifier-on-color` and use it for drying and humidifying.

There is a PR open for new humidifier more-info, but it's currently in review by the design team. Then, we can adjust the colors when we will have feedback on this PR for consistency.

For now, we can use the `--state-humidifier-on-color` for both actions.

Reply:
Done